packages/backend/src/handlers/actions/guild/saveGuildHandler.ts

```diff
@@ -149,4 +177,21 @@ const saveGuild = async (playerId: string, guildInfo: GuildInfoInput) => {
     guildId: guildInfo.uuid,
     discordMetadata: updatedMetadata,
   });
+
+  const { guild } = await client.GetGuild({ id: guildInfo.uuid });
+  return guild[0];
 };
+
+async function sendDiscordNotification(
+  guildInfo: GuildInfoInput,
+): Promise<void> {
```

Comment:
Given the name of the function, I think it makes sense for the channel id to be a param.

```suggestion
async function sendDiscordNotification(
  guildInfo: GuildInfoInput,
  channelId = Constants.METAFAM_DISCORD_CHAMPS_RING_CHANNEL_ID,
): Promise<void> {
```